cfme/tests/infrastructure/test_kubevirt.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
+from cfme.utils.update import update
+
+
+pytestmark = [
+    pytest.mark.provider([KubeVirtProvider], scope="module")
```

Comment:
as far as I understand this provider is available only in upstream. Where is restriction that this test has to be run only against upstream ? in yaml ?

Reply:
yes @izapolsk , yaml has it